CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # UNRELEASED
 
+-   Update matrix-rust-sdk to `#5d46b35d`, which includes:
+
+    -   Rename `VerificationLevel::PreviouslyVerified` to `VerificationViolation`
+        ([#4067](https://github.com/matrix-org/matrix-rust-sdk/pull/4067)
+
+    -   Rename `UserIdentity` to `OtherUserIdentity` and
+        Rename `UserIdentities` to `UserIdentity`
+        ([#4036](https://github.com/matrix-org/matrix-rust-sdk/pull/4036))
```

Comment:
If these are changes that affect the js-side API (I think they are?), can we instead express them in terms of breaking changes to the JS objects (which don't have `::` in their names, among other things)?

Reply:
Thanks, remove references to non-exported things in 220d6a2